### DISCUSSION TITLE
catridge create index problem

### DISCUSSION THREAD

**Discussion by tdudgeon** [rdkit/rdkit#5834] [Q&A]
```
postgres=# update molecule set mol = mol_from_smiles(smiles::cstring);
UPDATE 4846087
postgres=# create index molidx on molecule using gist(mol);
ERROR:  molFromPickle: invalid value in pickle
postgres=# select rdkit_version();
 rdkit_version 
---------------
 0.75.0
(1 row)

```
All mol objects were created OK, but creating the index fails.
How do I work out what is wrong?

**Comment by greglandrum** [comment]
Hi Tim. Which OS and version of postgresql are you using and how did you install the cartridge?

**Reply by tdudgeon** [reply]
Debian buster. RDKit 2021_03, Postgres 11.
Running in this container informaticsmatters/rdkit-cartridge-debian:Release_2021_03 that was built like this:
https://github.com/InformaticsMatters/docker-rdkit/blob/Release_2021_03/Dockerfile-cartridge-debian

I'm currently unable to build anything more recent due to problems with building the deb packages . Have spoke to @ptosco about that.

If I only insert 100 mols the index builds OK so it looks like its a problem with a  particualr mol.

**Reply by greglandrum** [reply]
Thanks. It would be really helpful if you could either isolate that particular molecule and send it or just send me the molecules so that I can try to do it.
I've seen reports of problems with the debian builds before but haven't been able to reproduce it myself yet in order to track down whether or not it's an RDKit problem (likely) or a build problem (less likely)

**Reply by ptosco** [reply]
@tdudgeon Hi Tim, can you remind me what the problem is? What is failing, exactly?

**Reply by tdudgeon** [reply]
@ptosco The building of the cartridge deb package stopped working (it was effectively empty). See my email from 24 Aug.

**Reply by tdudgeon** [reply]
@greglandrum I tracked down the offending molecules:

[Hf+5]
[V+6][Bi+2]

You may well ask what they are doing in a vendor catalog, but so be it!

**Reply by greglandrum** [reply]
Thanks Tim; I will take a look

**Reply by ptosco** [reply]
@tdudgeon I’ll take a look later, I promise; I had forgotten about this.

**Comment by ptosco** [comment]
@tdudgeon @greglandrum This is fixed by #5836.
